src/main/java/com/aws/greengrass/componentmanager/builtins/ArtifactDownloader.java

```diff
@@ -131,12 +135,13 @@ public File download() throws PackageDownloadException, IOException, Interrupted
                     offset.set(0);
                     messageDigest.reset();
                     throw new ArtifactChecksumMismatchException(
-                            "Integrity check for downloaded artifact failed. " + "Probably due to file corruption.");
+                            "Integrity check for downloaded artifact failed. Probably due to file corruption.",
```

Comment:
let's fix this error message, it has been confusing people.

`Probably due to the file changing after creating the component version`

Reply:
will update